### PR TITLE
Issue #170 Add apache 2.0 license

### LINKED ISSUE #170: Include a License
CRC is using Apache v2. Should the same be here?

Thanks.

### PR DESCRIPTION
**Pull request by anjannath**
But the generated bundle/image are not apache licensed

Fixes #170

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+Apache License
```
Comment (gbraad) [inline LICENSE:1]: indent wrong?

**Comment by anjannath** [comment]
@gbraad its there by default, even in the CRC repo, https://github.com/code-ready/crc/blob/master/LICENSE

**Comment by anjannath** [comment]
@gbraad Fixed the indentation, not sure from where we inherited it!

**Comment by mtarsel** [comment]
Sorry but if this is taken from CRC - shouldn't the name and year be added too? https://github.com/code-ready/crc/blob/master/LICENSE#L189 (same as this PR).

**Comment by mtarsel** [comment]
Disregard my last comment as that is only for adding the license to specific files.

**Pull request merged**